src/main/java/world/anhgelus/molehunt/Molehunt.java

```diff
@@ -60,25 +113,31 @@ public void onInitialize() {
         ).then(
                 literal("hide").executes(context -> {
                     timerVisibility.put(context.getSource().getPlayer(), false);
-                    context.getSource().sendFeedback(() -> Text.of("Hiding molehunt timer"), false);
+                    context.getSource().sendFeedback(() -> Text.translatable("commands.molehunt.timer.hide"), false);
                     return Command.SINGLE_SUCCESS;
                 })
         ));
-        command.then(literal("moles").requires(source -> (game != null) && game.isAMole(source.getPlayer())).executes(context -> {
-            context.getSource().sendFeedback(() -> Text.literal("List of moles: " + game.getMolesAsString()),false);
+        command.then(literal("moles").executes(context -> {
```

Comment:
Why did you remove the `.requires` ? It worked fine

Reply:
No, it removes the autocompletion

Reply:
For me it worked perfectly, unless using the function `game.hasStarted`. I guess maybe brigadier is inconsistent with this.

Reply:
Yeah ,Brigadier is inconsistent

Reply:
You added the game.hasStarted function in the isAMole function, that's the issue

Reply:
(I think)